src/bitcoind.cpp

```diff
@@ -5,14 +5,13 @@
 
 #include "chainparams.h"
 #include "clientversion.h"
-#include "rpcserver.h"
+#include "rpc/server.h"
 #include "init.h"
 #include "noui.h"
 #include "scheduler.h"
 #include "util.h"
 #include "httpserver.h"
 #include "httprpc.h"
-#include "rpcserver.h"
```

Comment:
Not sure why there was two `include`s here, but I removed the latter without ill effect.  I can put it back if necessary.